packages/twenty-front/src/modules/settings/developers/webhook/hooks/useAnalyticsTinybirdJwt.ts

```diff
@@ -0,0 +1,8 @@
+import { useRecoilValue } from 'recoil';
+
+import { currentUserState } from '@/auth/states/currentUserState';
+
+export const useAnalyticsTinybirdJwt = (): string | null | undefined => {
```

Comment:
can it be null

Reply:
yes it can be null when ANALYTICS is not enabled in the server